src/controllers/article.controllers.js

```diff
@@ -11,9 +12,11 @@ export const getArticles = async (req, res, next) => {
 };
 
 export const getArticleById = async (req, res, next) => {
+  const { id } = req.params;
   try {
-    const article = await Article.findById(req.params.id);
-    res.send(article);
+    const articleData = await Article.findById(id);
+    redisClient.setex(id, 3600, JSON.stringify(articleData));
```

Comment:
whats the second param here ? 3600

Reply:
it is how much time this cache's key (in this case id) is kept, i think in seconds, then it expires